aws-rds-dbparametergroup/src/main/java/software/amazon/rds/dbparametergroup/Translator.java

```diff
@@ -131,6 +136,18 @@ static ResourceModel translateFromDBParameterGroup(final DBParameterGroup dbPara
                 .collect(Collectors.toList());
     }
 
+    static String buildParameterGroupArn(final ResourceHandlerRequest<ResourceModel> request) {
```

Comment:
Let's please return an Arn and do a string conversion on the invocation side: it is always easier to convert it to string later on than to restore an Arn object.

Reply:
Make sense.